NEWS.md

```diff
@@ -1,25 +1,59 @@
-### v9.7.0 (2022-11-14)
-
-* Added new configuration option, `grpc.ignore_status_codes`, which can be used to select nonzero gRPC status codes to ignore and not report as errors.
-
-### v9.6.0 (2022-11-09)
-
-* Dropped support for `vision`, and instead only instrument `@hapi/vision`.
-
+### v9.7.1 (2022-12-05)
+
+* Reintroduced throttled reading of dependency tree at startup to prevent EMFILE issues.

```

Comment:
This sounds pretty jargony. Is there or should there be a clearer explanation of this? What dependency tree? What throttled reading?

Reply:
Maybe `Reintroduced throttling during reading of instrumented application's dependency tree during startup, to prevent EMFILE issues.` 

I don't know that we'd need to specify where and how we're throttling.